packages/@vue/cli/__tests__/invoke.spec.js

```diff
@@ -43,7 +43,25 @@ async function assertUpdates (project) {
 
   const eslintrc = parseJS(await project.read('.eslintrc.js'))
   expect(eslintrc).toEqual(Object.assign({}, baseESLintConfig, {
-    extends: ['plugin:vue/essential', '@vue/airbnb']
+    extends: ['plugin:vue/essential', '@vue/airbnb'],
+    rules: {
```

Comment:
IMO such duplication in test may not be the best choice. Better use `toMatchObject` and match only an essential subset of the config?

And a corresponding test can be added in the `@vue/cli-plugin-eslint` package to see if a vuex state mutation can pass the eslint check.

Reply:
ok :)